proxy/proxyUtils.js

```diff
@@ -138,6 +138,26 @@ function errorResponse(req, res, proxyRes, error) {
   return error_obj;
 }
 
+function auditEventObject(req, proxyResData) {
+  if (req.enableAudit) {
+    const data = JSON.parse(proxyResData.toString('utf8'));
+    let auditdata;
+    const reqType = auditEvent.auditEventObject.getEdataType(_.get(req, 'route.path'));
+    if (reqType.type === 'vote') {
```

Comment:
Writting if-else condition for each function is not good. I have shared the comments with @kamaleeswary.